assets/js/api_calls.js

```diff
@@ -1,20 +1,34 @@
 
-    $("#search").on("click", function() {
-        
-        yelpTags();
-        console.log("clicked");
-        
-    });
- 
-    function yelpTags() {
-        $(".loader").show();
-        console.log("search query run");
+    
+// each of these takes in an array of strings
```

Comment:
As I will be completing the UI/UX portion of the front-end
I'll make sure that I update these variables with the user input accordingly once they hit "search"